test/qctrl/test_qctrl.py

```diff
@@ -0,0 +1,394 @@
+            results_dict = {
+                "{0:02b}".format(key): value for key, value in result.quasi_dists[0].items()
+            }  # convert keys to bitstrings
+
+        ideal_result = {
+            key: val / shots for key, val in Statevector(bell_circuit).probabilities_dict().items()
+        }
+        fidelity = hellinger_fidelity(results_dict, ideal_result)
+
+        self.assertGreater(fidelity, FIDELITY_THRESHOLD)
+
+    @run_integration_test
+    def test_sampler_qctrl_ghz(self, service):
+        """Test qctrl small GHZ"""
+        shots = 1000
+        num_qubits = 5
+        ghz_circuit = QuantumCircuit(num_qubits)
+        ghz_circuit.h(0)
+        for i in range(num_qubits - 1):
+            ghz_circuit.cx(i, i + 1)
+
+        # Add measurements for the sampler
+        ghz_circuit_sampler = ghz_circuit.copy()
+        ghz_circuit_sampler.measure_active()
+
+        # Execute circuit in a session with sampler
+        with Session(service, backend=self.backend):
+            options = Options(resilience_level=1)
+            sampler = Sampler(options=options)
+
+            result = sampler.run(ghz_circuit_sampler, shots=shots).result()
+            results_dict = {
+                f"{{0:0{num_qubits}b}}".format(key): value
+                for key, value in result.quasi_dists[0].items()
+            }  # convert keys to bitstrings
+
+        ideal_result = {
+            key: val / shots for key, val in Statevector(ghz_circuit).probabilities_dict().items()
+        }
+        fidelity = hellinger_fidelity(results_dict, ideal_result)
+        self.assertGreater(fidelity, FIDELITY_THRESHOLD)
+
+    @run_integration_test
+    def test_sampler_qctrl_superposition(self, service):
+        """Test qctrl small superposition"""
+
+        shots = 1000
+        num_qubits = 5
+        superposition_circuit = QuantumCircuit(num_qubits)
+        superposition_circuit.h(range(num_qubits))
+
+        # Add measurements for the sampler
+        superposition_circuit_sampler = superposition_circuit.copy()
+        superposition_circuit_sampler.measure_active()
+
+        # Execute circuit in a session with sampler
+        with Session(service, backend=self.backend):
+            options = Options(resilience_level=1)
+            sampler = Sampler(options=options)
+
+            result = sampler.run(superposition_circuit_sampler, shots=shots).result()
+            results_dict = {
+                f"{{0:0{num_qubits}b}}".format(key): value
+                for key, value in result.quasi_dists[0].items()
+            }  # convert keys to bitstrings
+
+        ideal_result = {
+            key: val / shots
+            for key, val in Statevector(superposition_circuit).probabilities_dict().items()
+        }
+        fidelity = hellinger_fidelity(results_dict, ideal_result)
+        self.assertGreater(fidelity, FIDELITY_THRESHOLD)
+
+    @run_integration_test
+    def test_sampler_qctrl_conditional_states(self, service):
```

Comment:
These are "computational" states, not "conditional" states.
```suggestion
    def test_sampler_qctrl_computational_states(self, service):
```